### PR TITLE
Add job UUID to limiter debug logs in informer callbacks

### PR DESCRIPTION
**Pull request by DrJosh9000**
It's not relevant to how the limiter works currently, but interesting to see which jobs affect the limiter.

### CODE REVIEW

File: internal/controller/limiter/limiter.go [hunk 1]
```diff
@@ -130,14 +131,18 @@ func (l *MaxInFlight) OnAdd(obj any, inInitialList bool) {
...
 	// Before sync is finished: we're learning about existing jobs, so we should
 	// (try to) take tokens for unfinished jobs started by a previous controller.
 	// If it's added as already finished, no need to take a token for it.
 	// Otherwise, try to take one, but don't block (in case the stack was
 	// restarted with a different limit).
 	if !model.JobFinished(job) {
 		l.tryTakeToken("OnAdd")
-		l.logger.Debug("existing not-finished job discovered", zap.Int("tokens-available", len(l.tokenBucket)))
+		l.logger.Debug("existing not-finished job discovered",
+			zap.String("uuid", job.Labels[config.UUIDLabel]),
```
Comment (wolfeidau) [inline internal/controller/limiter/limiter.go:143]: should it be called job uuid?

**Review by wolfeidau — CHANGES_REQUESTED** [review]
Just one observation

File: internal/controller/limiter/limiter.go (hunk 1 above)
Comment (DrJosh9000) [reply internal/controller/limiter/limiter.go]: Done. For consistency I've changed it everywhere a job UUID field is logged.

**Review by wolfeidau — APPROVED** [review]
👍🏻 🚀

**Pull request merged**